docs/usage.md

```diff
@@ -0,0 +1,40 @@
+# Using Config Sync
+
+This document provides instructions on how to use Config Sync in a kubernetes
+cluster. This assumes Config Sync has already been [installed on the cluster](./installation.md).
+Installing Config Sync creates the Custom Resource Definitions (CRDs) which are
+used to configure Config Sync.
+
+## Config Sync CRDs
+
+Config Sync uses a concept of [Root repositories and Namespace repositories]. These
+concepts map to the RootSync and RepoSync CRDs, respectively.
+
+Installing Config Sync using Anthos Config Management includes the ConfigManagement CRD
```

Comment:
Add a link to the commercial version of installation on the GCP doc site (the OSS installation guide needs a similar link if it's not already there).

Reply:
done